### ISSUE TITLE
proxy settings allow https-only protocol support

### ISSUE THREAD

**Issue by yuriylesyuk** [apigee/apigee-config-maven-plugin#143]
[this is the counterpart of the https://github.com/apigee/apigee-deploy-maven-plugin/issues/178 issue]

As per documentation, "only https protocol is supported".

My case is an enterprise proxy with authentication that is only http.

When trying to use a plugin and proxy configuration in ~/.m2 config file, the plugin causes timeout that ends with 400.

Is there any way to add an http protocol proxy support?

Thank you

**Comment by ssvaidyanathan** [comment]
@yuriylesyuk - I have pushed a fix for this in a separate branch ([Issue143](https://github.com/apigee/apigee-config-maven-plugin/tree/Issue143)). Before I merge and release, can you test it for me please

Steps:
1. Proxy settings in your `~/.m2/settings.xml` file
2. Clone the repo and switch to Issue143 branch
3. Execute `mvn install` on the main project to create a 2.3.0-SNAPSHOT on your local .m2/repository
4. Update the `samples/EdgeConfig/shared-pom.xml` to use `2.3.0-SNAPSHOT` version (Line number 29)
5. `cd samples/EdgeConfig`
6. Export variables
```
export ORG=<APIGEE_ORG>
export ENV=<APIGEE_ENV>
export TOKEN=$(gcloud auth print-access-token)
```
7. Run `mvn clean apigee-config:targetservers -Ptest -Dorg=$ORG -Denv=$ENV -Dbearer=$TOKEN -Dapigee.config.dir=./resources/edge -Dapigee.config.options=update`

_NOTE: If you are using a different Apigee env other than "test" or "prod", just rename  one of the folders under "samples/EdgeConfig/resources/edge/env". For example if you are using "eval" environment, just update the "test" folder under "samples/EdgeConfig/resources/edge/env" to "eval"_

Let me know if it works with the proxy settings and report back if you see any issues

**Comment by ssvaidyanathan** [comment]
Fixed in v2.3.0
